akka-sample-kafka-to-sharding/build.sbt

```diff
@@ -0,0 +1,56 @@
+val AkkaVersion = "2.6.1+140-ea78b0af+20200117-0814"
```

Comment:
I wasn't able to pull this dep.  I assumed it might have been a snapshot, but it couldn't be resolved from the akka snapshots repo, so I published my own version of the upstream PR.

For my own future reference, where can I find such references?  Or was it assumed I would publish my own dep?

Reply:
This is this branch published locally https://github.com/akka/akka/pull/28211

Once it is merged we can update it to a nightly snapshot